webapp/src/Controller/BaseController.php

```diff
@@ -137,28 +137,19 @@ protected function saveEntity(
     }
 
     /**
-     * Perform the delete for the given entity
+     * Perform the delete for the given entities
      *
-     * @param Request                $request
-     * @param EntityManagerInterface $entityManager
-     * @param DOMJudgeService        $DOMJudgeService
-     * @param EventLogService        $eventLogService
-     * @param KernelInterface        $kernel
-     * @param                        $entity
-     * @param string                 $description
-     * @param string                 $redirectUrl
      * @throws DBALException
      * @throws NoResultException
      * @throws NonUniqueResultException
      */
-    protected function deleteEntity(
+    protected function deleteEntities(
         Request $request,
         EntityManagerInterface $entityManager,
         DOMJudgeService $DOMJudgeService,
         EventLogService $eventLogService,
         KernelInterface $kernel,
-        $entity,
-        string $description,
+        array $entities,
         string $redirectUrl
```

Comment:
Is this to prepare for multidelete?

Reply:
yes, as getting this right is hard so I rather do the backend logic first and thann have the frontend and the needed route for it.